src/language-js/embed.js

```diff
@@ -16,6 +17,38 @@ const {
   utils: { mapDoc, stripTrailingHardline }
 } = require("../doc");
 
+const cssPlaceholder = new Placeholder("prettier");
+const CSS_PROPERTY_PLACEHOLDER = new Placeholder("prettier").get(0);
+const CSS_EXTRA_SEMICOLON_MARK = new Placeholder("prettier").get(0);
+const placeholderPiecesToStringArray = pieces =>
+  pieces.map(({ isPlaceholder, string, placeholder }) =>
+    isPlaceholder ? placeholder : string
+  );
+
+const cleanCSS = string =>
+  string
+    // preserve `prettier-ignore` comments
+    .replace(/\/\*\s*prettier-ignore\s*\*\//g, "_")
+    .replace(/\/\/\s*prettier-ignore/g, "_")
+    // remove block comments
+    .replace(/\/\*[\s\S]*?\*\//g, "")
+    // remove quoted strings
+    .replace(/(['"]).*?(\1)/g, "_")
+    // remove strings in parentheses
+    .replace(/\(.*?\)/g, "_")
+    // remove inline comments
+    .replace(/\/\/.*/g, "");
```

Comment:
@lydell I renamed this function and replace all of then to `_`, It really doesn't matter :)

Reply:
Looks like RegExp hell, they will ever kill us :smile: 

Don't forget some of css-in-js support Sass/Less/etc (for example https://github.com/4Catalyzer/astroturf) and using regexp potentially can break code, especially quoted strings/strings in parentheses, I'm sure there are problems